app/pages/sections/order-items/catalogue-solutions/delete/confirmation/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "{{solutionName}} deleted from {{orderId}}",
+    "description": "This Catalogue Solution has been deleted from all the Service Recipients in your order along with any of its Additional Services.",
+    "continueButtonText": "Continue"
+}
```

Comment:
The file should end with a trailing new line.

Reply:
resolved